docs/etl/README.template.md

```diff
@@ -168,12 +168,15 @@ It is important that the first transformer is a `MultiInputTransformer` when hav
 ### Example-7
 
 This example illustrates the use of `TransformerNC`.
+This example illustrates the use of `Transformer` in a non conuming context.
 The job here is to join the two extracted dataframes - an employees dataframe and a birthdays dataframe.
-But, before the birthdays can be joined onto the employees, the employees dataframe require a transformation step.
+But, before the birthdays can be join onto the employees, the employees dataframe require a transformation step.
```

Comment:
can be joined is correct. Actually later in the line, "require" should be "requires"